evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -68,6 +68,7 @@ class IterativeAlgorithm(AutoMLAlgorithm):
         verbose (boolean): Whether or not to display logging information regarding pipeline building. Defaults to False.
         exclude_featurizers (list[str]): A list of featurizer components to exclude from the pipelines built by IterativeAlgorithm.
             Valid options are "DatetimeFeaturizer", "EmailFeaturizer", "URLFeaturizer", "NaturalLanguageFeaturizer", "TimeSeriesFeaturizer"
+        excluded_model_families (list[ModelFamily]): A list of model families to exclude from the estimators used when building pipelines.
```

Comment:
Unrelated request - could you add allowed_model_families to this docstring? Unless it's here and I'm just totally not seeing it

Reply:
It's there, just out of order. I'll update.